Tests/LinuxMain.swift

```diff
@@ -0,0 +1,17 @@
+import XCTest
+
+import CoreTests
+import FixtureTests
+import PerformanceTests
+import ProjectSpecTests
+import XcodeGenKitTests
+
+var tests = [XCTestCaseEntry]()
+tests += CoreTests.__allTests()
+tests += FixtureTests.__allTests()
+tests += PerformanceTests.__allTests()
+tests += ProjectSpecTests.__allTests()
+tests += XcodeGenKitTests.__allTests()
+
+XCTMain(tests)
```

Comment:
We could potentially replace the contents of this file with 
```
// LinuxMain.swift
fatalError("Run the tests with `swift test --enable-test-discovery`.")
```
And then delete all the XCTestManifests, as per https://oleb.net/2020/swift-test-discovery/